### DISCUSSION TITLE
The number of samples in MD17 dataset is not the same as described in the documentation

### DISCUSSION THREAD

**Discussion by yusowa0716** [pyg-team/pytorch_geometric#4334] [General]
The number of samples in MD17 dataset is not the same as described in the documentation.

See the example:
![image](https://user-images.githubusercontent.com/52536919/159942834-8cc03146-7098-4821-a284-6d7263bcd911.png)
![image](https://user-images.githubusercontent.com/52536919/159943024-e9730920-bf53-4421-b8f9-deb4b13f2b54.png)

I checked the original _npz_ file and the number doesn't match the documentation either, I'm curious if the original author of MD17 cleaned the data? Or something else?

**Comment by rusty1s** [comment]
Thanks for reporting. I updated the documentation accordingly, see https://github.com/pyg-team/pytorch_geometric/commit/b5d752d0a8dd2df5e2e2d8fca502163c48eb9f2b.

**Reply by yusowa0716** [reply]
Thank you for your response :)
